mxcube3/ui/components/SampleQueue/CurrentTree.js

```diff
@@ -36,7 +36,8 @@ export default class CurrentTree extends React.Component {
 
   nextSample() {
     if (this.props.todoList[0]) {
-      this.props.runSample(this.props.todoList[0]);
+        this.props.mount(this.props.sampleList[this.props.todoList[0]]);
```

Comment:
You should not need to mount the sample before running a sample, its taken care of by the sample
queue entry. This already works here for us without the extra mount step so it should be working for you
to.